astropy/table/table.py

```diff
@@ -595,6 +595,11 @@ class Table:
 
     # Unified I/O read and write methods from .connect
     read = UnifiedReadWriteMethod(TableRead)
+    """
+    Read and parse a data table and return as a Table.
```

Comment:
I prefer this way. The one with `#:` is not obvious to me that it is supposed to be rendered.

Reply:
Agreed.
I would prefer if the descriptor's docstring were rendered... but that appears to be a sphinx related issue. If it can be solved in Astropy-sphinx, that would obviate this good workaround.

Perhaps #11554 can be transferred to Astropy-sphinx and not closed by this PR.

Reply:
This method is sphinx specific and means that the docstring rendered by sphinx will be different from what one sees only. Does something like `read.__doc__ = """..."""` work as well? That would make it the same in all places.

Reply:
I think @nstarman tried that already and it didn't work.

Reply:
It don't recall it working when I first raised the issue.
In particular, it didn't work when I did it in ``__set_name__``

```python
def __set_name__(self, parent, name):
    setattr(getattr(parent, name), __doc__, self.__doc__)
```